base/v0_2_exp/validate_test.go

```diff
@@ -134,3 +134,25 @@ func TestValidateResource(t *testing.T) {
 		}
 	}
 }
+
+func TestValidateTree(t *testing.T) {
+	tests := []struct {
+		in  Tree
+		out error
+	}{
+		{
+			in:  Tree{},
+			out: ErrTreeNoLocal,
+		},
+	}
+
+	for i, test := range tests {
+		actual := test.in.Validate(path.New("yaml"))
+		expected := report.Report{}
+		expected.AddOnError(path.New("yaml"), test.out)
+
+		if !reflect.DeepEqual(actual, expected) {
+			t.Errorf("#%d: expected %+v got %+v", i, expected, actual)
```

Comment:
Should this not be using the `%v` formatting with calls to `format` on both `expected` & `actual` similar to what was done in one of the previous commits?